### DISCUSSION TITLE
Ender 3 V2 with Creality Sprite Pro Extruder printing problems

### DISCUSSION THREAD

**Discussion by cfdeangelis** [mriscoc/Ender3V2S1#1341] [Q&A] [answered]
Having problems with my Creality Sprite Pro extruder on Ender 3 V2 w/ 4.2.7 board.  I used the configurator correctly (I'm pretty sure), but when trying to print, I can't even get a first layer down, just a bunch of stringing/balling up at the nozzle.  I've correctly set my z-offset, but what else might I be missing?  Thanks for any help!

![image](https://github.com/user-attachments/assets/4e58c410-f295-425a-a7a6-38eba5aca4f0)
![IMG_3829](https://github.com/user-attachments/assets/e05c6a5c-c64a-4a4f-a90e-39a3f26626a1)

**Comment by cfdeangelis** [comment, marked as answer]
After reading some stuff online, I adjusted my e-steps, which left me at a value somewhere around 425. I had originally thought this was quite high, but Creality actually recommends a value of 424.9 for the Sprite extruder, so this value is correct.  It may be worth adjusting the Configurator (https://github.com/mriscoc/Special_Configurations) to use this setting as default when selecting T13, as this corresponds to the Sprite Extruder Pro temperature sensor.

**Comment by D-J-ARES** [comment]
https://photos.app.goo.gl/jyLBzWhc7y4szDKv5
